### ISSUE TITLE
Fork this project

### ISSUE THREAD

**Issue by tlivings** [krakenjs/hapi-openapi#212]
I think it makes sense to potentially fork this project to maintainers willing to continue work on it and redirect users in the readme to the new maintained project. 

Afaik the Kraken team doesn't actively maintain this project and I haven't had time in a while to keep up with it.

Thoughts @dcharbonnier ?

**Comment by dcharbonnier** [comment]
I agree, if this can't stay in the same namespace with more people that have access to issues, and merge requests.
But I won't maintain it alone, no point if nobody check the PR at least.

**Comment by dcharbonnier** [comment]
ping @tpburch @alferpal @sommestad

**Comment by alferpal** [comment]
Thanks for the ping.

In the end we went with a different approach to what we needed and didn't ended up using this.

Kind of built something a bit similar on top of ajv, more tailored to what we needed
